connectors/src/lib/models/intercom.ts

```diff
@@ -271,7 +271,7 @@ export class IntercomArticle extends Model<
 
   declare state: "draft" | "published";
 
-  declare lastUpsertedTs?: Date;
+  declare lastUpsertedTs?: Date | null;
```

Comment:
Do we need that, I don't think we've used it elsewhere but I might be wrong?

Reply:
Fiendly bump

Reply:
Yes, it allows me to know when I upserted the datasource and when I upserted the article object only